lib/stdlib/doc/src/zip.xml

```diff
@@ -368,8 +368,11 @@
           <seeerl marker="erl_tar"><c>erl_tar</c></seeerl>.</p>
         <p><c><anno>FileList</anno></c> is a list of files, with paths relative 
           to the current directory, which are stored with this path in the
-          archive. Files can also be specified with data in binaries
-          to create an archive directly from data.</p>
+          archive. File system operations are perform to read the file metadata
```

Comment:
shouldn't this be "File system operations are perform*ed*"?

Reply:
```suggestion
          archive. File system operations are performed to read the file metadata
```